specviz/plugins/loader_wizard/loader_wizard.py

```diff
@@ -478,6 +568,15 @@ def _clear_unit_status(self):
         self.ui.label_unit_status.setStyleSheet('')
 
     def save_loader_check(self):
+        """
+        Check wizard for valid units and loader name before saving loader to file.
+
+        Returns
+        -------
+        bool:
```

Comment:
This should be `<something> : bool`

Reply:
You can just write ``bool`` and forget about the colon and the name - this is allowed by Numpydoc since return values don't have to have a name.